library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -88,6 +91,7 @@ interface ZoomApi {
      * @see panY
      * @see AbsolutePan
      */
+    @Target(AnnotationTarget.FIELD, AnnotationTarget.FUNCTION, AnnotationTarget.LOCAL_VARIABLE, AnnotationTarget.PROPERTY, AnnotationTarget.VALUE_PARAMETER)
     @Retention(AnnotationRetention.SOURCE)
     annotation class ScaledPan
```

Comment:
I regret using these name, something like AbsoluteCoord would have been better. But we will change in another version

Reply:
Those names still confuse me :grin: 
They are also kind of useless since the IDE seems to do nothing to check these.
Even when passing wrong values to a function with annotated parameters there is no warning.
That's another reason why I opted to create data objects for them.

Reply:
Ah, without them it would be worse I think... There should be an android annotation to tell the IDE that these two annotations exclude each other, but I don't know how.

Reply:
Yes they do help indeed! But not to the extend I'd like them to :stuck_out_tongue: